packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Analytics": "Analytics",
+  "Content Manager": "Innehållshantering",
+  "Content Type Builder": "Innehållstyp Byggare",
+  "Email": "E-post",
+  "Files Upload": "Uppladdning av filer",
+  "HomePage.notification.newsLetter.success": "Prenumererade på nyhetsbrevet",
+  "New entry": "Nytt inlägg",
+  "Password": "Lösenord",
+  "Provider": "Leverantör",
+  "ResetPasswordToken": "Återställ lösenordstoken",
+  "Role": "Roll",
+  "Roles & Permissions": "Roller och behörigheter",
+  "Settings Manager": "Inställningshanterare",
+  "Username": "Användarnamn",
+  "Users": "Användare",
+  "Users & Permissions": "Användare och behörigheter,
+  "app.components.BlockLink.code": "Kodsexempel",
+  "app.components.BlockLink.code.content": "Lär dig genom att testa verkliga projekt utvecklade samhället.",
```

Comment:
"Lär dig genom att testa verkliga projekt utvecklade av communityn." You could use "Nätförening" for a more "proper" swedish, but I wouldn't use that term here...

Reply:
> "Lär dig genom att testa verkliga projekt utvecklade av communityn." You could use "Nätförening" for a more "proper" swedish, but I wouldn't use that term here...

Changed to"communityn"